lib/src/rules/prefer_relative_imports.dart

```diff
@@ -85,6 +66,7 @@ class _Visitor extends SimpleAstVisitor<void> {
     String importUri = node?.uri?.stringValue;
     if (importUri == null) return false;
 
+    // todo (pq): is there a better way to do this?
```

Comment:
Consider `node.uriSource.uri`.

Reply:
Unfortunately that doesn't seem to work for what we're trying to do (though maybe I'm missing something).  The rub is that:

```dart
import 'dummy.dart';
```    

reports a `package` URI scheme but that confuses our test for whether the import is package qualified.